browser/css/notebookbar.css

```diff
@@ -298,7 +295,8 @@
 }
 
 .unotoolbutton.notebookbar.has-label:not(.inline) img {
-	width: 24px !important;
+	width: var(--icon-size-big);
+	height: var(--icon-size-big);
```

Comment:
This seems to be unrelated to this commit since we are changing value from 24 to 32px